12.10/sortStation.c

```diff
@@ -0,0 +1,177 @@
+        }
+        if (isOperator(string[i])) {
+            while (!isEmpty(stack) && isOperator(top(stack)) && isPriorityOfSecondOperatorHigherOrEqual(string[i], top(stack))) {
+                char secondOperator = pop(stack, errorCode);
+                if (*errorCode) {
+                    printf("Попаю пустой стэк, вероятно не правильная арифметическая запись\n");
+                    deleteStack(stack);
+                    return NULL;
+                }
+                if (indexInOutputExpression + 1 >= maxLineSize) {
+                    printf("При преобразовании максимальное количество элементов в выражении превышено\n");
+                    *errorCode = 1;
+                    deleteStack(stack);
+                    return  NULL;
+                }
+                outputExpressionInPrefixForm[indexInOutputExpression] = secondOperator;
+                outputExpressionInPrefixForm[++indexInOutputExpression] = ' ';
+                ++indexInOutputExpression;
+            }
+            push(stack, string[i]);
+            continue;
+        }
+        if (string[i] == '(') {
+            push(stack, string[i]);
+            continue;
+        }
+        if (string[i] == ')') {
+            //Добавить в топ проверку на пустоту стэка
+            while (top(stack) != '(') {
+                char topValue = pop(stack, errorCode);
+                if (*errorCode) {
+                    printf("Попаю пустой стэк, вероятно не правильная арифметическая запись(проблема со скобками)\n");
+                    deleteStack(stack);
+                    return NULL;
+                }
+                if (indexInOutputExpression + 1 >= maxLineSize) {
+                    printf("При преобразовании максимальное количество элементов в выражении превышено\n");
+                    *errorCode = 1;
+                    deleteStack(stack);
+                    return  NULL;
+                }
+                outputExpressionInPrefixForm[indexInOutputExpression] = topValue;
+                outputExpressionInPrefixForm[++indexInOutputExpression] = ' ';
+                ++indexInOutputExpression;
+            }
+            pop(stack, errorCode);
+            if (*errorCode) {
+                printf("Попаю пустой стэк, вероятно не правильная арифметическая запись\n");
+                deleteStack(stack);
+                return NULL;
+            }
+            continue;
+        }
+        *errorCode = 1;
+        printf("Неизвестный символ в выражении\n");
+        deleteStack(stack);
+        return NULL;
+    }
+    while (!isEmpty(stack)) {
+        char topValue = pop(stack, errorCode);
+        if (*errorCode) {
+            printf("Попаю пустой стэк, вероятно не правильная арифметическая запись\n");
+            deleteStack(stack);
+            return NULL;
+        }
+        if (topValue == '(') {
+            *errorCode = 1;
+            printf("Попаю пустой стэк, вероятно не правильная арифметическая запись(проблема со скобками)\n");
```

Comment:
```suggestion
            printf("Попаю пустой стэк, вероятно не правильная арифметическая запись (проблема со скобками)\n");
```
Русский язык надо знать лучше, чем языки программирования :)